### PR TITLE
CDRW-686 Fake OpenId Authorisation Server Url in Mock Server for initial client redirection

### PR DESCRIPTION
**Pull request by daniel-kuras**
Exposes a mock OpenID Config endpoint. This endpoint is found here `/openid/config/:id`. It provides links to:
* `/token` endpoint.
* `/authorize` endpoint.

Also, updated the ASPSP payloads to include a `OpenIDConfigEndPointUri`. This points to the above path.

### CODE REVIEW

File: lib/ob-directory.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+const env = require('env-var');
 
-const aspsp = (name, fapiFinancialId) => ({
+const openId = env.get('OPENID_CONFIG_ENDPOINT_URL').asString();
```
Comment (robmckinnon) [inline lib/ob-directory.js]: We could instead construct this URL from the `OPENID_ASPSP_AUTH_HOST`, eg:

`` const openId = `${env.get('OPENID_ASPSP_AUTH_HOST').asString()}/openid/config`; ``

Then we'd have one less env var to explain and define. :)

File: test/aspsp-test.js [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+const openIdConfigUrl = 'http://localhost/openid/config';
 
 describe('/scim/v2/OBAccountPaymentServiceProviders', () => {
+  let apspsServer;
+  let server;
+
+  before(() => {
+    apspsServer = proxyquire('../lib/ob-directory.js', {
+      'env-var': env.mock({
+        OPENID_CONFIG_ENDPOINT_URL: openIdConfigUrl,
```
Comment (robmckinnon) [inline test/aspsp-test.js]: If we made the suggestion above ^ then this line would change to `OPENID_ASPSP_AUTH_HOST`.

**Comment by robmckinnon** [comment]
Do we want to update readme as we go? Or update readme as a batch job at end of sprint?

**Comment by ezodude** [comment]
Prob update README as we go. I'll have a look at it.

File: lib/ob-directory.js (hunk 1 above)
Comment (ezodude) [reply lib/ob-directory.js]: As I understand it, these two concepts `OPENID_CONFIG_ENDPOINT_URL` and `OPENID_ASPSP_AUTH_HOST` are semantically different. And, I am not sure about hardcoding the path `/openid/config`.

Comment (robmckinnon) [reply lib/ob-directory.js]: Ok, understood.

**Comment by ezodude** [comment]
Having looked at the README, I think beyond the scope of this PR. Especially, if we want small succinct PRs.

**Pull request merged**